cdc/owner/metrics.go

```diff
@@ -65,13 +69,34 @@ var (
 			Name:      "status",
 			Help:      "The status of changefeeds",
 		}, []string{"changefeed"})
+	changefeedTickDuration = prometheus.NewHistogramVec(
```

Comment:
maybe add a screenshot after the metric applied make things more clear, the dashboard template will be added later ?